src/generated/todo/getTodosSaga.js

```diff
@@ -0,0 +1,75 @@
+// This file is generated
+
+import { takeEvery } from 'redux-saga';
+import { call, put } from 'redux-saga/effects';
+
+// FIXME - implement proper api
+function api(aPathParam, aQueryParam) {
+  return new Promise((resolve) => {
+    // Use superagent
```

Comment:
superagent 2!
https://github.com/visionmedia/superagent/blob/v2.0.0/docs/index.md#promise-and-generator-support